spec/merch_calendar/retail_calendar_spec.rb

```diff
@@ -57,6 +60,35 @@
       expect(subject.end_of_quarter(2019, 4)).to eq Date.new(2020, 2, 1)
     end
   end
+  
+  describe "#quarter" do
+    it "returns the correct date" do
+      expect(subject.quarter(5)).to eq 2
+      expect(subject.quarter(7)).to eq 3
+      expect(subject.quarter(2)).to eq 1
+      expect(subject.quarter(11)).to eq 4
+    end
+  end
+  
+  describe "#season" do
+    context "returns Spring/Summer from its merch_month" do
```

Comment:
`context "for merch months in the Spring/Summer season"` more accurately describes the context.

Saying "something returns something" is only valid with an `it "returns..."`.  

For all of these rspec things I'm pointing out, a good guideline is to read the test out loud to yourself.  It should make sense in plain English.

Reply:
👍